### DISCUSSION TITLE
sway version

### DISCUSSION THREAD

**Discussion by revkarol** [regolith-linux/regolith-desktop#833] [Q&A] [answered]
Hi,

Is it possible to try the sway version yet?   

(I'm on pop os 22.04 if that matters).   Not afraid to compile something :)

**Comment by cfsmp3** [comment, marked as answer]
Yes, I've been using it for months.

I believe it's in the unstable branch now, just update your apt source.

**Reply by revkarol** [reply]
I had to use the 22.10 version, and despite the fact that pop os doesn't have a 22.10 version, it still seems to work :)   

so now my `/etc/apt/sources.list.d/regolith.list` looks like this:

`deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/unstable-ubuntu-kinetic-amd64 kinetic main`

Thanks @cfsmp3
